scss/config/_colors.scss

```diff
@@ -22,14 +22,14 @@
 ///   Accoutrement Color
 /// @example scss
 ///   // @preview color-palette
-///   @include herman-add-colors('brand-colors', $brand-colors);
+///   @include accoutrement-add('colors', 'brand-colors', $brand-colors);
 $brand-colors: (
   'brand-orange': hsl(24, 100%, 39%),
   'brand-blue': hsl(195, 85%, 35%),
   'brand-pink': hsl(330, 85%, 48%),
 );
 
-@include herman-add-colors('brand-colors', $brand-colors);
+@include accoutrement-add('colors', 'brand-colors', $brand-colors);
```

Comment:
I wonder if the 'name' can somehow be pulled from the $mapname?

The only time we don't use the same exact naming here is for the font family names. So we use 'sans' but the map is called $font-sans. But maybe that is just our own use case?

Reply:
I agree it would be nice, but I can't figure out any way to do that. Sass isn't able to get the name of a variable - only the value - and we need both Sass and Herman to agree on the name.

This isn't really about changing the name or keeping it the same, it's just about telling Sass what the name is, so that Herman can find it. Which is ugly, but I don't have a better idea.